sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/SortAggregateExec.scala

```diff
@@ -49,7 +49,7 @@ case class SortAggregateExec(
   override def producedAttributes: AttributeSet =
     AttributeSet(aggregateAttributes) ++
       AttributeSet(resultExpressions.diff(groupingExpressions).map(_.toAttribute)) ++
-      AttributeSet(aggregateBufferAttributes)
+      AttributeSet(aggregateBufferAttributes) ++ super.producedAttributes
```

Comment:
(Tihs is not related to this PR though) `producedAttributes` looks the same between the three aggregate implementations, so could we move `producedAttributes` into `BaseAggregateExec`?

Reply:
Good idea!